index.js

```diff
@@ -93,6 +94,10 @@ function buildResponseMessage(prDetails) {
     }
     if (!isValidPullRequestBody(prDetails.body)) {
         message += '1. PR Description\n';
+        if (process.env.ENABLE_KEYWORD_CHECKER != undefined
+            && process.env.ENABLE_KEYWORD_CHECKER.toLowerCase() === "true") {
```

Comment:
No, I meant have a single environment variable that encodes a JSON object, instead of having an environment variable for every setting.